src/main/java/org/openstreetmap/atlas/geography/atlas/pbf/store/PbfMemoryStore.java

```diff
@@ -502,9 +503,25 @@ public boolean isAtlasPoint(final long identifier)
 
     public boolean isAtlasPoint(final Node node)
     {
-        // Each node will have a last edit time tag
-        return node.getTags().size() > AtlasTag.TAGS_FROM_OSM.size()
-                || containsNodeInRelations(node.getId());
+        if (containsNodeInRelations(node.getId()))
+        {
+            return true;
+        }
+        // Tags from OSM are the tags that all the nodes will have
+        if (node.getTags().size() > AtlasTag.TAGS_FROM_OSM.size())
```

Comment:
This logic behind needs some explanation and seems easy to break. I'd suggest adding some comments and explaining why it makes a node an Atlas point.

Reply:
Created https://github.com/osmlab/atlas/issues/11 as well. Again for this PR I did not want to change the logic here, but that deserves a refactor.
I will add more comments as well in the mean time.

Reply:
The design was any OSM node with artificial tag will be promoted as an Atlas Point.
So here it is checking if the OSM node has artificial tag other than those default tags from OSM (like LastEditTimeTag etc.).